examples/tree_2d_dgsem/elixir_linearized_euler_convergence.jl

```diff
@@ -0,0 +1,61 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linearized euler equations
+
+equations = LinearizedEulerEquations2D(rho=1.0, lambda=1.0)
+
+initial_condition = initial_condition_convergence_test
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+solver = DGSEM(polydeg=3, surface_flux=flux_lax_friedrichs)
```

Comment:
Technically, you can use the LLF (Lax-Friedrichs-Flux) as surface flux. But, as @gregorgassner suggested, it is better to use an upwinding flux here. I think it is not that of a big effort to implement a numerical upwinding flux and use this instead of LLF.

Reply:
The name we use in Trixi.jl for this is `flux_godunov`